src/components/WalletAssetsHeadline.vue

```diff
@@ -0,0 +1,240 @@
+          <s-radio class="radio-btn" v-for="(filter, index) in filterOptions" :key="index" :label="filter">{{
+            filter
+          }}</s-radio>
+        </s-radio-group>
+        <s-divider class="wallet-assets-divider__popover" />
+        <div class="wallet-assets-headline__switch">
+          <s-switch v-model="onlyVerifiedAssets" :disabled="verifiedOnlySwitch" />
+          <span>{{ t('filter.verifiedOnly') }}</span>
+        </div>
+        <div class="wallet-assets-headline__switch">
+          <s-switch v-model="zeroBalanceAssets" :disabled="zeroBalanceSwitch" />
+          <span>{{ t('filter.zeroBalance') }}</span>
+        </div>
+        <div class="wallet-assets-headline__button" slot="reference">
+          {{ t('filter.show').toUpperCase() }}:
+          <span class="wallet-assets-headline__button-option">{{ chosenOptionText.toUpperCase() }}</span>
+          <s-icon class="wallet-assets-headline__button-icon" name="basic-settings-24" size="14px" />
+        </div>
+      </el-popover>
+    </div>
+    <s-divider class="wallet-assets-divider" />
+  </div>
+</template>
+
+<script lang="ts">
+import { Component, Mixins, Prop } from 'vue-property-decorator';
+
+import { mutation, state } from '@/store/decorators';
+import FormattedAmount from './FormattedAmount.vue';
+import TranslationMixin from './mixins/TranslationMixin';
+import LoadingMixin from './mixins/LoadingMixin';
+import { WalletAssetFilters, WalletFilteringOptions } from '../consts';
+
+@Component({
+  components: {
+    FormattedAmount,
+  },
+})
+export default class WalletAssetsHeadline extends Mixins(TranslationMixin, LoadingMixin) {
+  @Prop({ default: '0', type: String }) readonly assetsFiatAmount!: string;
+
+  @state.settings.filters filters!: WalletAssetFilters;
+  @mutation.settings.setFilterOptions private setFilterOptions!: (filter: WalletAssetFilters) => void;
+
+  verifiedOnlySwitch = false;
+  zeroBalanceSwitch = false;
+
+  get onlyVerifiedAssets(): boolean {
+    return this.filters.verifiedOnly;
+  }
+
+  set onlyVerifiedAssets(value: boolean) {
+    this.updateFilters('verifiedOnly', value);
+  }
+
+  get zeroBalanceAssets(): boolean {
+    return this.filters.zeroBalance;
+  }
+
+  set zeroBalanceAssets(value: boolean) {
+    this.updateFilters('zeroBalance', value);
+  }
+
+  get selectedFilter(): string {
+    return this.filters.option;
+  }
+
+  set selectedFilter(value) {
+    this.updateFilters('option', value);
+  }
+
+  updateFilters(key: string, value: string | boolean): void {
+    const filters = {
+      ...this.filters,
+      [key]: value,
+    };
+
+    this.setFilterOptions(filters);
+    this.$emit('filter-assets', filters);
+  }
+
+  get filterOptions(): Array<string> {
+    return Object.values(WalletFilteringOptions);
+  }
+
+  updated(): void {
+    if (this.filters.option === WalletFilteringOptions.NFT) {
+      // disable verified only switch as there are no whitelisted NFTs.
```

Comment:
But maybe they will be in future? Do we need this code btw, if the result for user will be the same at that moment

Reply:
I don't think we will have whitelisted NFTs in foreseeable future.
As for disabled switch, if you turn it on, it will not show any nft tokens in nft "tab" as all nfts are non-whitelisted, it just does not make any sense to leave it as is